foam_wrapper/foam_wrapper.py

```diff
@@ -0,0 +1,228 @@
+""" foam_wrapper module
+
+Wrapper module for OpenFOAM using pythonFlu -python interface
+NOTE: at a moment pythonFlu supports only up to 2.0.1 -version of OpenFOAM
+
+"""
+
+from Foam import ref, man
+from simphony.cuds.mesh import Mesh, Point, Face, Edge, Cell
+#import inspect
```

Comment:
We should probably remove commented out code before we merge this PR with master